tests/conftest.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+
+# Inferentia fixtures
+ENCODER_ARCHITECTURES = [
+    "albert",
+    "bert",
+    "camembert",
+    "convbert",
+    "distilbert",
+    "electra",
+    "flaubert",
+    "mobilebert",
+    "mpnet",
+    "roberta",
+    "roformer",
+    "xlm",
+    "roberta",
+]
+DECODER_ARCHITECTURES = ["gpt2", "llama"]
+DIFFUSER_ARCHITECTURES = ["stable-diffusion", "stable-diffusion-xl"]
```

Comment:
```suggestion
DIFFUSION_ARCHITECTURES = ["stable-diffusion", "stable-diffusion-xl"]
```
diffusers is the library name, I think diffusion could be more appropriate.